tests/typecheck/managers/test_managers.yml

```diff
@@ -48,7 +48,7 @@
                 class Base(Generic[_T]):
                     def __init__(self, model_cls: Type[_T]):
                         self.model_cls = model_cls
-                        reveal_type(self.model_cls._default_manager)  # N: Revealed type is "django.db.models.manager.BaseManager[django.db.models.base.Model]"
+                        reveal_type(self.model_cls._default_manager)  # N: Revealed type is "django.db.models.manager.BaseManager[_T`1]"
```

Comment:
Let's also test `._base_manager` of some `Model`, so we can ensure that `_T` is correct.

Reply:
I'm super unfamiliar with these types of tests. I hope the new test makes sense!